skins/spicedao.scss

```
@@ -0,0 +1,10 @@
+.spicedao {
+    --primary-color: #000;
+    --bg-color: #1201ff;
+    --link-color: white;
+    --heading-color: #000;
+    --border-color: #000;
+    --header-bg: #cf0;
+    --block-bg: transparent;
+    --text-color: #fff
```

You forgot to close this line.
```suggestion
    --text-color: #fff;
```